### PR TITLE
Implemented new koans

### PR DESCRIPTION
**Pull request by Etxben**
Implemented new koans, triangleProject and triangleProject2.
Fix links on README.md to python koans.

### CODE REVIEW

File: koans/TriangleProject2Koans.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace PhpKoans;
+require_once 'Triangle.php';
```
Comment (AsierAlba10) [inline koans/TriangleProject2Koans.php:4]: ¿This is really neccesary?, you can better use this `use PhpKoans\Triangle`

File: koans/TriangleProject2Koans.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+{
+
+    private Triangle $triangle;
+
+    /**
+     * @setUp
+     */
+    protected function setUp() : void
+    {
+        $this->triangle = new Triangle();
```
Comment (AsierAlba10) [inline koans/TriangleProject2Koans.php:18]: You forgot to put `parent::setUp();` at the beggining of the setUp().
If we don't put this, the tests will fail.

File: koans/TriangleProjectKoans.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+{
+    
+    private Triangle $triangle;
+
+    /**
+     * @setUp
+     */
+    protected function setUp() : void
+    {
+        $this->triangle = new Triangle();
```
Comment (AsierAlba10) [inline koans/TriangleProjectKoans.php:18]: The same like in the other class :)

**Review by AsierAlba10 — APPROVED** [review]
Nice work! Just a couple of little things :)

**Comment by Etxben** [comment]
Try another way.

**Pull request closed without merging**